bpf/lib/conntrack.h

```diff
@@ -615,18 +616,19 @@ static __always_inline int ct_lookup4(const void *map,
 					    &identifier, 2) < 0)
 				return DROP_CT_INVALID_HDR;
 
-			tuple->sport = 0;
 			tuple->dport = 0;
 
+			/* convert from unsigned char to unsigned short considering byteorder(little-endian) */
+			tuple->sport = (__u16)(type << 8);
```

Comment:
Why is casting not enough to convert from 1 byte to 2 bytes? What's special about the other 8 bits? That would be good to clarify in this comment.

Reply:
This is because of byte order. In the little-endian case, for example 2byte data "AB" convert to "BA", so we should shifting the "type" not just casting. 
I commented about it in the line above, but it may not be the best explanation. So, you mean it should be more specific?